src/main/java/com/github/vkorobkov/jfixtures/config/structure/util/TableMatcher.java

```diff
@@ -10,7 +10,6 @@
 
 @FunctionalInterface
 public interface TableMatcher {
-
```

Comment:
Let's not add style only changes into classes with are not related to the given feature